docs/api/ProxyAgent.md

```diff
@@ -74,6 +76,26 @@ for await (const data of body) {
 }
 ```
 
+#### Example - Basic Proxy Request with authentication
+
+```js
+import { setGlobalDispatcher, request, ProxyAgent } from 'undici';
+
+const proxyAgent = new ProxyAgent({
+  uri: 'my.proxy.server',
+  token 'Bearer xxxx'
```

Comment:
```suggestion
  token: 'Bearer xxxx'
```

Reply:
@cola119
Sorry! I have fixed it.
https://github.com/nodejs/undici/pull/1705/commits/fb7a9c43eb4ecc05f87f8805693d32ded2cf8e47